storage/migrations/17_chainvis_views_not_materialized.go

```diff
@@ -0,0 +1,145 @@
+package migrations
```

Comment:
What is the motivation for calling this before the migration previously holding this number (17)? An alternative might be:
- Leave 17 (multisig transactions) and 18 (drops unused tables) alone. 
- Add migration 19 with the desired changes from here.

Reply:
As discussed in chat we will take these migration as is and flatten our schema https://github.com/filecoin-project/sentinel-visor/issues/225 in a follow on.